lib/public/views/Runs/formatRunDuration.js

```diff
@@ -27,55 +28,45 @@ export const formatRunDuration = (run, raw = false) => {
     const { runDuration, timeTrgStart, timeTrgEnd, timeO2Start, timeO2End } = run;
     if (runDuration === null || runDuration === undefined) {
         return '-';
-    } else if (timeTrgEnd || timeO2End) {
-        return `${formatDuration(runDuration)}${raw ? '' : getAsteriks(run)}`;
-    } else {
-        const timeStart = timeTrgStart === null || timeTrgStart === undefined
-            ? timeO2Start
-            : timeTrgStart;
-        let ret = {
-            text: 'RUNNING',
-            classes: 'success',
-        };
+    }
 
-        // If trigger start is more than 48 hours ago, consider that AliECS crashed without ending the run
-        if (Date.now() - timeStart > MAX_RUN_DURATION) {
-            ret = {
-                text: 'UNKNOWN',
-                classes: 'danger',
-            };
-        }
+    const missingTimeTrgStart = timeTrgStart === null || timeTrgStart === undefined;
+    const missingTimeTrgEnd = timeTrgEnd === null || timeTrgEnd === undefined;
+
+    if (timeTrgEnd || timeO2End) {
+        let warningPopover = null;
 
-        if (raw) {
-            return ret.text;
+        if (missingTimeTrgStart && missingTimeTrgEnd) {
+            warningPopover = 'Duration based on o2 start AND stop because of missing trigger information';
+        } else if (missingTimeTrgStart) {
+            warningPopover = 'Duration based on o2 start because of missing trigger information';
```

Comment:
I think in the 2 scenarios of missing only one of the trigger times, the message should be adapted accordingly.
`missing trigger information` when only trigger start or stop is missing is not correct

Reply:
Updated